vaadin-simple-grid-filter/src/main/java/software/xdev/vaadin/model/CustomizationDegree.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © 2024 XDEV Software (https://xdev.software)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package software.xdev.vaadin.model;
+
+public enum CustomizationDegree
```

Comment:
A short documentation would be helpful, could be based on: 
"When editing a condition, the developer can specify how much of the condition is editable."

Reply:
Added doc.